ups/eupspkg.cfg.sh

```diff
@@ -3,5 +3,5 @@ TAP_PACKAGE=1
 
 build()
 {
-    CFLAGS=-I${LIBYAML_DIR}/include python setup.py --with-libyaml build
+    CFLAGS="-I${LIBYAML_DIR}/include -L${LIBYAML_DIR}/lib" python setup.py --with-libyaml build
```

Comment:
LDFLAGS seems to work fine for the container build, and is definitely better.  Updated PR and  re-running Jenkins...